interop/grpc_testing/test.proto

```diff
@@ -214,10 +214,17 @@ message LoadBalancerStatsRequest {
 }
 
 message LoadBalancerStatsResponse {
+  message RPCsByPeer {
```

Comment:
I believe the proto style here is for this to be camel-cased: RpcsByPeer.

Reply:
Done